vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -788,6 +788,34 @@ function Get-SystemDriveDiskInfo {
     }
 }
 
+function Register-ExpandVolumeTask {
+    $toolsDir = "c:\aks-tools\expand-volume"
+    if (!(Test-Path -Path $toolsDir)) {
+        New-Item -ItemType Directory -Path $toolsDir | Out-Null
+    }
+    $taskScript = 
+@'
+$expandVolumeLogPath = 'c:\aks-tools\expand-volume\expand-volume.log'
+$diskpartScriptPath = 'c:\aks-tools\expand-volume\diskpart.script'
+if (-not (Test-Path -Path $diskpartScriptPath)) { # Only run once
```

Comment:
Should we check the log file instead of the script itself?

Reply:
remove the task directly